pkg/appconsts/versioned_consts_test.go

```diff
@@ -13,39 +13,89 @@ import (
 )
 
 func TestSubtreeRootThreshold(t *testing.T) {
-	type testCase struct {
-		version uint64
-		want    int
-	}
-	testCases := []testCase{
-		{version: v1.Version, want: v1.SubtreeRootThreshold},
-		{version: v2.Version, want: v2.SubtreeRootThreshold},
-		{version: testground.Version, want: testground.SubtreeRootThreshold},
+	testCases := []struct {
+		version  uint64
+		expected int
+	}{
+		{
+			version:  v1.Version,
+			expected: v1.SubtreeRootThreshold,
+		},
+		{
+			version:  v2.Version,
+			expected: v2.SubtreeRootThreshold,
+		},
+		{
+			version:  testground.Version,
+			expected: testground.SubtreeRootThreshold,
+		},
 	}
+
 	for _, tc := range testCases {
 		name := fmt.Sprintf("version %v", tc.version)
 		t.Run(name, func(t *testing.T) {
 			got := appconsts.SubtreeRootThreshold(tc.version)
-			assert.Equal(t, tc.want, got)
+			require.Equal(t, tc.expected, got)
 		})
 	}
 }
 
-func TestSquareSizeUpperBound(t *testing.T) {
-	type testCase struct {
-		version uint64
-		want    int
+func TestGlobalMinGasPrice(t *testing.T) {
+	testCases := []struct {
+		version  uint64
+		expected float64
+		expErr   bool
+	}{
+		{
+			version:  v2.Version,
+			expected: v2.GlobalMinGasPrice,
+			expErr:   false,
+		},
+		{
+			version:  v1.Version,
+			expected: 0,
+			expErr:   true,
+		},
 	}
-	testCases := []testCase{
-		{version: v1.Version, want: v1.SquareSizeUpperBound},
-		{version: v2.Version, want: v2.SquareSizeUpperBound},
-		{version: testground.Version, want: testground.SquareSizeUpperBound},
+
+	for _, tc := range testCases {
+		name := fmt.Sprintf("version %v", tc.version)
+		t.Run(name, func(t *testing.T) {
+			got, err := appconsts.GlobalMinGasPrice(tc.version)
+			if tc.expErr {
+				require.Error(t, err)
+			} else {
+				require.NoError(t, err)
+				require.Equal(t, tc.expected, got)
```

Comment:
I think this line is meant to be under the `if` block.

Reply:
I thought it was sufficient enough to just check that the error was thrown. if you think that it's better if I take that line out of the `if`  block I'm more than happy to!

Reply:
I previously thought that `tc.expected` referred to the expected error. No modification is necessary, the current version is correct.